public/kibi_timeline_vis_params.html

```diff
@@ -34,6 +34,13 @@
                 Show highlighted terms
                 <kbn-info info="Show highlighted terms in event label."></kbn-info>
              </label>
+            <label>
+              <input type="checkbox"
+                name="highlightFirstLabelInstance"
+                ng-model="group.invertFirstLabelInstance">
+              Emphasize first label instance
+              <kbn-info info="Cut thru the clutter and make unique label values standout. When checked, the first instance of a label will be displayed with inverted colors."></kbn-info>
```

Comment:
I think I'd prefer through instead of thru http://www.differencebetween.net/language/words-language/difference-between-thru-and-through/